cleanrl/atari/v_c51_atari.py

```diff
@@ -0,0 +1,283 @@
+#C51 Paper: https://arxiv.org/pdf/1707.06887.pdf
+#Paper Implementation: https://github.com/vwxyzjn/cleanrl/blob/master/cleanrl/c51_atari.py
+#Vectorized : https://github.com/vwxyzjn/cleanrl/blob/experimental-vdqn/cleanrl/atari/vdqn_atari.py
+
+import argparse
+import os
+import random
+import time
+from distutils.util import strtobool
+import gym
+
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.optim as optim
+from stable_baselines3.common.atari_wrappers import (
+    ClipRewardEnv,
+    EpisodicLifeEnv,
+    FireResetEnv,
+    MaxAndSkipEnv,
+    NoopResetEnv,
+)
+from torch.utils.tensorboard import SummaryWriter
+
+def parse_args():
+    parser = argparse.ArgumentParser(description='Vectorized-C51 agent')
+    # Common arguments
+    parser.add_argument('--exp-name', type=str, default=os.path.basename(__file__).rstrip(".py"),

```

Comment:
add `# fmt: off` as a new line before `parser.add....`

Reply:
Sure , i would like to know  the purpose of this comment # fmt: off , # fmt: on

Reply:
This is for formatting. After doing it try doing `black vc51_.py` (Auto-formatting). Also would you mind renaming the file to `vc51_atari.py`? Sorry for the trouble.

Reply:
I think the full command I use is 

```
isort . --skip wandb
autoflake -r --exclude wandb --in-place --remove-unused-variables --remove-all-unused-imports .
black -l 127 --exclude wandb .
```

Also, would you mind putting this file under a folder called `vectorized_architecture`?

Reply:
Costa , I will make the required  changes. Keep me posted of all the required changes

Reply:
Pushed the formatted code